lib/cfg-grammar.y

```diff
@@ -903,7 +903,7 @@ template_content_inner
           CHECK_ERROR_GERROR(log_template_compile(last_template, $3, &error), @3, error, "Error compiling template");
           free($3);
 
-          CHECK_ERROR_GERROR(log_template_set_type_hint(last_template, $1, &error), @1, error, "Error setting the template type-hint");
+          CHECK_ERROR_GERROR(log_template_set_type_hint(last_template, $1, &error), @1, error, "Error setting the template type-hint \"%s\"", $1);
```

Comment:
Note: I was concerned, what if we get a NULL or invalid pointer if the string parameter is not provided. Answer: It is handled earlier by the parser as a missing parameter. So we got at least an "" empty string at this point.